examples/policies/write_to_file_events.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: aquasecurity.github.io/v1beta1
+kind: TraceePolicy
+metadata:
+  name: write
+  annotations:
+    # https://aquasecurity.github.io/tracee/latest/docs/events/builtin/syscalls/write/
+    description: traces write events across all scopes
+spec:
+  scope: 
+    - global
+  rules:
+    - event: write
```

Comment:
This event on its own doesn't cover all file writes. There are some other syscalls which may trigger a file write (e.g. writev) so it needs to be clarified in the description or update the policy with all syscalls that may trigger a file write

Reply:
aha, thank you for clarifying -- will update it